content/docs/apps/node.md

```diff
@@ -21,6 +21,14 @@ Deploying a basic node.js application to Cloud Foundry is relatively uncomplicat
     ```
 
 - Avoid using the `command:` manifest key to start your application, as this functionality [may be removed](https://github.com/cloudfoundry/nodejs-buildpack/pull/11#issuecomment-67666273). The aforementioned npm start script is preferred.
+
+- Vendor your dependencies.  Cloud Foundry [documentation](https://docs.cloudfoundry.org/buildpacks/node/index.html#vendoring) specifies that you should do a `npm install` to install vendored dependencies and pushing your entire application with `cf push`. This is especially
+important for large applications as Cloud Foundry limits the size of an app upload to `1GB` -
+see [Deploying a Large Application](https://docs.cloudfoundry.org/devguide/deploy-apps/large-app-deploy.html) for more information.
+
+- Do not put test harnesses or transpilers in your dependencies object. View the npm
+[documentation](https://docs.npmjs.com/files/package.json#dependencies) for more on this.
```

Comment:
In this section it'd be good to include a section pointing out that `NODE_ENV` should always be set up to `production` in order to only have `dependencies: { /**/ }` be downloaded otherwise `devDependencies: { /**/ }` will also be downloaded which could affect disk space and memory quotas.